### ISSUE TITLE
docs redesign: A mobile text size setting affects only some text

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#2884]
### Current

Example screenshot from a Pixel phone, 412x892 resolution, portrait view, with 200% text size. The Chrome browser setting is Accessibility > Text Scaling

This setting enlarges the "Page Contents" list. It does not enlarge the banner or the way finding breadcrumbs.

If this browser setting is replied upon for conformance, then this issue fails WCAG 1.4.4 Resize Text. Or, if this setting is not relied upon for conformance, it's still an issue that affects users of the accessibility setting, making it difficult to read the text.

![Screenshot as described above](https://user-images.githubusercontent.com/3124805/209431138-344821b6-ef70-48be-a730-b1b1ac6ce2a2.png)

### Proposed

Allow the browser text size setting to apply to all text in the page.

**Comment by kfranqueiro** [comment]
I just tested this and it appears to have been resolved. All text on the page resizes when text size is changed, and I see rem or em units used in the locations that were specified in the original issue.
